src/ui/Quickview/Quickview.ts

```diff
@@ -212,7 +230,19 @@ export class Quickview extends Component {
         }
         return DomUtils.getBasicLoadingAnimation();
       }
-    )
+    ),
+
+    /**
+     * Specifies where to position the tooltip compared to the element
+     * Default value is `bottom`. Other values are `auto`, `top`, `left`, `right`.
```

Comment:
I will add @fbeaudoincoveo to review the text.

One suggestion I have though is, looking at the other jsdoc examples, we usually place the default value on the last line of the doc instead of wrapped in the middle of the main text. Maybe something like:

```
     /**
     * Specifies where to position the tooltip compared to the element
     * Values are `auto`, `top`, `left`, `right`, `bottom`.
     * You can also append these variations: `-start`, `-end` which correspond to the alignment (by default they are centered)
     *
     * Default value is `bottom`.
     *
     * **Example:**
     * `top-start` tooltip is on top of the `Quickview` button, aligned on the left (start)
     */
```

Reply:
Suggestions:
- `Placement` type -> `ValidTooltipPlacement`
- Document the `ValidTooltipPlacement` type as follows:
```
/**
* The allowed [`Quickview`]{@link Quickview} [`tooltipPlacement`]{@link Quickview.options.tooltipPlacement} option values. The `-start` and `-bottom` variations indicate relative alignment. Horizontally (`top`, `bottom`), `-start` means _left_ and `end` means _right_; vertically (`left`, `right`), `-start` means _top_ and `-end` means _bottom_. No variation means _centered_.
*/
```
- Modify the `tooltipPlacement` description as follows:
```
/**
* Specifies the emplacement of the tooltip in relation to the `Quickview` HTML element.
*
* **Example:**
* > If you set this option to `top-start`, the tooltip will appear on top of the `Quickview` button, aligned on the left.
*
* Default value is `bottom`.
*/ 
```